superflore/generators/bitbake/run.py

```diff
@@ -93,7 +93,7 @@ def main():
             warn_msg =\
                 'removing existing symlink "./recipes-ros-{0}"'.format(x)
             RepoInstance.warn(warn_msg)
-        except:
+        except Exception:
```

Comment:
What sort of exceptions are ok here vs not ok? This satisfies the linter but still doesn't resolve the underlying question